tests/vue/fixtures/test/index.test.ts

```diff
@@ -0,0 +1,10 @@
+import { expect, test } from '@rstest/core';
+import { mount } from '@vue/test-utils';
+// @ts-expect-error
```

Comment:
We can remove this, for IDE, we could use Vue official extension. For type check, which we're not using now, we could use `vue-tsc`.